packages/docs/docs/fhir-datastore/patient-deduplication/patient-deduplication.md

```diff
@@ -0,0 +1,193 @@
+
+### Terminology
+
+Before we get started, let's define some terminology.
+
+In this guide, we will call the output of your deduplication pipeline the **target system**. Each input system that contributes patient data will be called a **source system.**
+
+Most pipelines make a copy of each patient record from the source system into the target system. We will call these copies **source records.** The final, combined patient record will be called the **master record**.
+
+### Pipeline
+
+While deduplication pipelines can take many forms, there three primary operations in every pipeline:
+
+1. **Ingestion:** Copying data from source systems into the target system to create source records.
+2. **Matching:** For each patient record, find all other records that could are potential matches.
+3. **Merging:** Merge all the information into a single record to serve as the source of truth.
+
+There are many different ways to implement this pipeline, but there are a few key decisions that you will need to make when choosing an architecture:
+
+- Whether you are building a batch vs. incremental deduplication pipeline
+- Which matching rules to use
+- How to merge matching patients into a single record
+
+### Decision Factors
+
+We'll discuss each of these in depth. The most important factors to consider when making these decisions are:
+
+- **Prevalence of duplicates:** Making estimate of your duplication rate, both _within_ and _between_ source systems, will help you determine the complexity needed in your deduplication pipeline. Some applications have rare duplication rates (e.g. < 1%) while others have frequent duplicates (10-50%). Still others have very bi-modal duplication patterns (e.g 99% of records have 0 duplicates, 1% of records have 5+ duplicates).
+
+- **Duration of source systems:** Some pipelines are built to ingest patient data from short-lived source systems, with the target system quickly replacing them as the source of truth. Examples include migrating from legacy systems into medplum, ingestion from one-time data sources. In contrast, some source systems are long lived or permanent. This is the case when deduplicating patients from multiple healthcare providers, or when building pipelines from Health Information Exchanges (HIEs).
+
+- **Frequency of unmerging patients:** No deduplication system will be 100% perfect. Certain architectural choices make it easier or harder to unmerge patient records once merged, and the right choice for you will depend on the cost of an incorrect merge and the frequency with which you will unmerge records.
+
+- **Downstream application:** The right deduplication architecture will ultimately depend on how the deduplicated data will be used. Target systems that serve web and mobile applications will have different priorities than that systems that serve as data aggregators. Services that target providers will have different considerations than a system that will also support patient login.
```

Comment:
Instead of sentence three, I'd add an example.  e.g. For example, some systems support patient login and a merge will show a whole new set of data to a user, merging for this use case should be distinct from merging for population health or insurance billing purposes.